src/command/cmd_funcs.c

```diff
@@ -9195,7 +9195,11 @@ cmd_avatar(ProfWin* window, const char* const command, gchar** args)
         return TRUE;
     }
 
-    if (g_strcmp0(args[0], "get") == 0) {
+    if (g_strcmp0(args[0], "set") == 0) {
+        if (avatar_set(args[1])) {
```

Comment:
I would suggest to add the gdkpixbuf lib as an optional dependency.
At this point we should add an ifdef and in case Profanity is compiled without the pixbuf lib print some informative message (like done in OMEMO etc cases with ligsignal).